dev-util/edb-debugger/edb-debugger-1.0.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="edb is a cross platform x86/x86-64 debugger, inspired by Ollydbg"
+HOMEPAGE="https://github.com/eteran/edb-debugger"
+
+SRC_URI="https://github.com/eteran/edb-debugger/releases/download/${PV}/edb-debugger-${PV}.tgz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="graphviz"
+
+S="${WORKDIR}/edb-debugger-${PV}"
+
+RDEPEND="
+	dev-libs/capstone
+	graphviz? ( media-gfx/graphviz )
+	dev-qt/qtwidgets:5
+	dev-qt/qtxml:5
+	dev-qt/qtxmlpatterns:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtconcurrent:5
+	dev-qt/qtgui:5
+	dev-qt/qtcore:5
+	"
+DEPEND="
+	dev-libs/boost
+	virtual/pkgconfig
+	${RDEPEND}"
+
+src_prepare(){
+	#Make the desktop's entries somewhat better
+	sed -i -e 's/GenericName=edb debugger/GenericName=Evan\x27s Debugger/' edb.desktop || die
+	sed -i -e 's/Comment=edb debugger/Comment=edb is a cross platform x86\/x86-64 debugger/' edb.desktop || die
```

Comment:
Could you submit this upstream, please?

Reply:
[Done.](https://github.com/eteran/edb-debugger/pull/657) 

I am afraid this will only help us for live and future versions. If this is what you had in mind, I am perfectly fine.

Reply:
Ya. The point is not to have to carry this forever.

Reply:
I'm sorry that I haven't proofread your changes. Left comment on your PR upstream. Long story short, GenericName should be just 'Debugger'.